code/modules/research/techweb/all_nodes.dm

```diff
@@ -505,6 +505,9 @@
 		"mesons_prescription",
 		"multi_cell_charger",
 		"tray_goggles_prescription",
+		"plumbing_chem",
+		"plumbing_eng",
+		"plumbing_sci"
```

Comment:
You should alphabetize these since you're touching it anyway

Reply:
Do you mean the whole list or just the changes?

Reply:
The tech ids are in alphabetical order (separated because it was nicer in code for modularisation)